src/Transaction/BaseFirestoreTransactionRepository.spec.ts

```diff
@@ -510,9 +564,19 @@ describe('BaseFirestoreTransactionRepository', () => {
       it('should correctly parse dates', async () => {
         await bandRepository.runTransaction(async tran => {
           const pt = await tran.findById('porcupine-tree');
-          const { releaseDate } = await pt.albums.findById('deadwing');
+          const albumsRef = pt.albums;
+          if (!albumsRef) {
+            throw new Error('Albums should not have an undefined value');
+          }
+
+          const album = await albumsRef.findById('fear-blank-planet');
+          if (album === null) {
+            throw new Error('Album should not have a null value');
+          }
+          const { releaseDate } = album;
+
           expect(releaseDate).toBeInstanceOf(Date);
-          expect(releaseDate.toISOString()).toEqual('2005-03-25T00:00:00.000Z');
+          expect(releaseDate.toISOString()).toEqual('2007-04-16T00:00:00.000Z');
```

Comment:
Had to change the date because tests were breaking for some reason.

Reply:
That's weird, nothing has changed in the test fixtures file 🤔